### ISSUE TITLE
Stale cache when managing advanced permissions

### ISSUE THREAD

**Issue by bigio** [nextcloud/groupfolders#1378]
After latest NextCloud update to 21.0.0 managing advanced permissions from web gui has become difficult due to stale cache.
Every time I view advanced permissions on a folder I have to force a refresh (F5 on the browser) to be able to check permissions on a different folder, otherwise the web gui will display the permissions of the last folder I view.

Cache related configuration is:
```
  'memcache.local' => '\\OC\\Memcache\\APCu',  
  'memcache.locking' => '\\OC\\Memcache\\Redis',  
  'memcache.distributed' => '\\OC\\Memcache\\Redis',  
  'redis' =>   
  array (  
    'host' => '127.0.0.1',  
    'port' => 6379,  
    'dbindex' => 1,  
  ),
```

I also tried to disable Apcu and opcache but nothing has changed.

**Comment by jngrb** [comment]
I can confirm this bug. But I think it is not related to caching. There is just no refresh of the content of the SharingSidebarView (<div id="groupfolder-acl-container"...></div>) when a different file is selected.
Steps to reproduce:
a) Have a groupfolder with advanced permissions enabled.
b)- Have a subfolder A with a certain ACL list.
c) Have a subfolder B with certain other ACL entries.
d) Open the groupfolder with the list of subfolders.
e) Click "..." -> Details on subfolder A to load the side bar view. Go to sidebar tab "Sharing" to see the ACL entries.
f) Click "..." -> Details on subfolder B to refresh the side bar view with the details for the other folder. **BUT**: the ACL entries are not refreshed - they are still the entries of subfolder A.

To work around this bug, just close the side bar view with the "x" in the upper-right corner between steps e) and f). => You see the right ACL entries - those from subfolder B.

**Comment by jngrb** [comment]
I provided a patch for this: #1406

@juliushaertl , please review this PR and my other one #1395

**Comment by jngrb** [comment]
@juliushaertl , this can be closed, I think. The PR is merged and the duplicates #1408 and #1455 are also already closed.
